com.github.inercia.k3x.json

```
@@ -0,0 +1,124 @@
+{
+  "app-id": "com.github.inercia.k3x",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.34",
```

Update runtime to 3.36
```suggestion
  "runtime-version": "3.36",
```